toolbox/build/venv-requirements-ansible-2.9.txt

```diff
@@ -0,0 +1 @@
+ansible>=2.9.1,<2.10
```

Comment:
All looks good, i'm just wondering how to test the lower-bound supported OpenStack SDK as well. We could add the `openstacksdk` condition here. Or create another job for it in a separate PR?

Overall +1 that we can merge after squashing.

Reply:
I can add a new PR for that